formatTest/typeCheckedTests/expected_output/attributes.re

```diff
@@ -237,7 +237,10 @@ class boxB ('a) (init: 'a) =
 
 /* To be able to put an attribute on just the return in that case, use
  * parens. */
-[@onBoxC x; y]
+[@onBoxC
```

Comment:
this one is a little awkward

Reply:
At first, I found it weird but later made sense. Any structure_item inside an attribute should be treated like a structure_item without an attribute...

I can roll it back thought.

Reply:
that does make sense. happy to keep it as is